editors/emacs/dune

```diff
@@ -10,6 +10,7 @@
  (package lambdapi)
  (section share_root)
  (files
+  (highlight.el as emacs/site-lisp/highlight.el)
```

Comment:
The dune file has been removed in `master`, it has been replaced by a makefile in ea571d801a139f3fc3281742ff66d37294d703a7.

Reply:
Oh, didn't see that! Thanks.

Reply:
For the sake of testing I had to modify this dune file again, but I assume it'll not be a problem when merging...

Reply:
Yes I see, I think you will only have to delete this file when merging.